### DISCUSSION TITLE
Contrato por hora/mês ou serviço com cliente?

### DISCUSSION THREAD

**Discussion by fabiochiquezi** [frontendbr/forum#2028] [Perguntas e Respostas]
Fala pessoal,

Seguinte tenho uma empresa que quer fechar negócio (software web) comigo, mas estou com problemas em relação a definir um contrato de serviço. O principal ponto é, cobrar por mês ou por serviço? Como vocês fazem e o que acham melhor?

Algumas questões que penso a respeito:

- O cliente pode não ter uma ideia bem definida do que vai ser o software
- As vezes o que ele pensa ser simples é complexo
- Geralmente tem retrabalho depois de experimentar o software e ver que precisa de alterações
- Não dá pra eu saber o tempo exato que vai levar o projeto

Acredito eu que a melhor estratégia é cobrar um valor fixo por mês, fazer sprints semanalmente definindo tarefas do que vai ser entregue e construindo com ele o software, testando e desenvolvendo tarefas sprint a sprint.

No entanto, por não estar acostumado com esse tipo de serviço, ele prefere que eu passe um valor fixo de quanto vai ficar o software. Mas acho que terei problemas com isso, pq não dá pra eu definir o tempo que vai levar esse projeto.

Vocês já passaram por isso? Qual a experiência de vocês?

**Comment by francysreymer** [comment]
Valor fixo por software é furada, porque é praticamente impossível
determinar todos os detalhes que até muitas vezes só são pensados no dia a
dia.
Cobra por valor de hora. Daí vc determina quantas horas tem em um mês e
multiplica pelo valor.
Aí vc determina as entregas mensais ou quinzenais com base na sua hora.

>

**Comment by gabriel-hahn** [comment]
Olá Fábio! Realmente, cobrar valor fixo pelo software pronto pode ser uma furada, principalmente se muitos requisitos estão em aberto e/ou o contrato não determinar EXATAMENTE o que, quando e como deve ser entregue - o que é bem difícil, já como o @francysreymer comentou, muitas coisas e problemas só são pensados durante o desenvolvimento e amadurecimento da ideia. Na minha opinião, esses modelos de valor fixo por software funcionam muito bem para aplicações menores (parece não ser o seu caso), landpages ou algo com escopo e contrato (importante) bem definido.

Uma sugestão seria cobrar por hora mesmo, ficando mais flexível para a empresa analisar seu rendimento e o quanto está pagando por cada _feature_ entregue, assim como você vai ter uma segurança maior caso algo sair fora do controle, não absorvendo tanto a responsabilidade se o cliente solicitar várias alterações não planejadas anteriormente. Resumidamente, mesmo que o cliente mude de ideia durante o projeto, você vai receber a mais pelas alterações o/

Particularmente, é a opinião que tenho, mas gostaria de ler mais respostas do pessoal também hehe abraço.
